Policies/Kubernetes/allowed-external-ips/examples-good/azurepolicy.assignment.parameters.json

```diff
@@ -0,0 +1,6 @@
+{
+  "parameters": {
+    "excludedNamespaces": { "value": [ "kube-system", "azure-arc" ] },
+    "allowedExternalIPs": { "value": [ "203.0.113.0"] }
```

Comment:
do we need a default one?

Reply:
I don't think so. We don't know the IPs that users can use.

Reply:
can we leave this blank then?

Reply:
Yep. I referred to other policies. Some of them leave the blank which means that all the values are not allowed. 
Here the line is just an example. The real value in our policy is located at `azurepolicy.json`
```
"allowedExternalIPs": {
        "type": "Array",
        "metadata": {
          "displayName": "Allowed External IPs",
          "description": "List of External IPs that services are allowed to use. Empty array means all external IPs are disallowed."
        },
        "defaultValue": []
      }
```
```
"allowedExternalIPs": "[parameters('allowedExternalIPs')]"
```
Now the default value is blank.

Reply:
ah ok i see, looks like this file is just an example.